### ISSUE TITLE
Expose Metadata

### ISSUE THREAD

**Issue by endocrimes** [dantoml/nomad-plugin-firecracker#4]
Firecracker has support for a metadata k/v api, we should use this to inject expected Env values into the VM.

**Comment by plasticine** [comment]
@dantoml I’ll probably have a stab at an implementation for this if you’d be open to a PR?

**Comment by endocrimes** [comment]
@plasticine Definitely! - I'm hoping to clean up the way we start firecracker to support jailer later this week, which should be independent of most of metadata wiring 👍
